vue/src/components/SideBar.vue

```diff
@@ -183,6 +184,14 @@ const satelliteLoadingColor = computed(() => {
           <span class="text-caption">{{ state.appVersion }}</span>
         </div>
       </v-row>
+      <v-row
+        dense
+        align="center"
+        justify="center"
+        class="mb-3"
+      >
+        <upload-model-run />
```

Comment:
on success should a signal be emitted that indicates the modelrun listing should be reloaded?

Reply:
I added a commit that refreshes the performers and regions. Is there another bit of state that should be refreshed when the model run is uploaded?